.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Feature
+about: Suggest a new feature for elrond-go
+title: "[FEAT]"
+labels: enhancement
+assignees: ''
+
+---
+
+<!--
+Tell us what newer versions of elrond-go shoud include. Add relevant information about why you need the new feature for and why does the existing feature set is not enough. Also, please explain in detailed steps how the feature should work
```

Comment:
I know i have wrote this. Can you please fix the typo "shoud" ?

Reply:
:+1: